pkgs/development/libraries/qt-5/5.8/qtwebkit/0004-icucore-darwin.patch

```diff
@@ -0,0 +1,11 @@
+--- qtwebkit-opensource-src-5.8.0.orig/Source/WTF/WTF.pri
++++ qtwebkit-opensource-src-5.8.0/Source/WTF/WTF.pri
+@@ -12,7 +12,7 @@
+     # Mac OS does ship libicu but not the associated header files.
+     # Therefore WebKit provides adequate header files.
+     INCLUDEPATH = $${ROOT_WEBKIT_DIR}/Source/WTF/icu $$INCLUDEPATH
+-    LIBS += -licucore
++    LIBS += /usr/lib/libicucore.dylib
```

Comment:
Thx for the hint, but can you elaborate what impact `__impureHostDeps` has on buildInputs and the like?